Apps/Sandcastle/gallery/Particle System Tails.html

```diff
@@ -0,0 +1,228 @@
+var particlesOffset = new Cesium.Cartesian3(-8.950115473940969, 34.852766731753945, -30.235411095432937);
+var cameraLocation = Cesium.Cartesian3.add(planePosition, particlesOffset, new Cesium.Cartesian3());
+var recenterCamera = function() {
+    viewer.camera.lookAt(cameraLocation, new Cesium.Cartesian3(170, -170, 170));
+};
+recenterCamera();
+
+// Draw particle image to a canvas
+var particleCanvas;
+function getImage() {
+    if (!Cesium.defined(particleCanvas)) {
+        particleCanvas = document.createElement('canvas');
+        particleCanvas.width = 20;
+        particleCanvas.height = 20;
+        var context2D = particleCanvas.getContext('2d');
+        context2D.beginPath();
+        context2D.arc(8, 8, 8, 0, Cesium.Math.TWO_PI, true);
+        context2D.closePath();
+        context2D.fillStyle = 'rgb(255, 255, 255)';
+        context2D.fill();
+    }
+    return particleCanvas;
+}
+
+// Add plane to scene
+var hpr = new Cesium.HeadingPitchRoll(0.0, Cesium.Math.PI_OVER_TWO, 0.0);
+var orientation = Cesium.Transforms.headingPitchRollQuaternion(planePosition, hpr);
+var entity = viewer.entities.add({
+    model : {
+        uri : '../../SampleData/models/CesiumAir/Cesium_Air.gltf',
+        scale : 3.5
+    },
+    position : planePosition,
+    orientation : orientation
+});
+
+// creating particles model matrix
+var transl = Cesium.Matrix4.fromTranslation(particlesOffset, new Cesium.Matrix4());
+var translPosition = Cesium.Matrix4.fromTranslation(planePosition, new Cesium.Matrix4());
+var particlesModelMatrix = Cesium.Matrix4.multiplyTransformation(translPosition, transl, new Cesium.Matrix4());
+
+// creating the particle systems
+var rocketOptions = {
+    numberOfSystems : 50.0,
+    iterationOffset :  0.1,
+    cartographicStep : 0.000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        minimumRed : 1.0,
+        green : 0.5,
+        minimumBlue : 0.05,
+        alpha : 1.0
+    }, {
+        red : 0.9,
+        minimumGreen : 0.6,
+        minimumBlue : 0.01,
+        alpha : 1.0
+    }, {
+        red : 0.8,
+        green : 0.05,
+        minimumBlue : 0.09,
+        alpha : 1.0
+    }, {
+        minimumRed : 1,
+        minimumGreen : 0.05,
+        blue : 0.09,
+        alpha : 1.0
+    }]
+};
+
+var cometOptions = {
+    numberOfSystems : 100.0,
+    iterationOffset : 0.003,
+    cartographicStep : 0.0000001,
+    baseRadius : 0.0005,
+
+    colorOptions : [{
+        red : 0.6,
+        green : 0.6,
+        blue : 0.6,
+        alpha : 1.0
+    }, {
+        red : 0.6,
+        green : 0.6,
+        blue : 0.9,
+        alpha : 0.9
+    }, {
+        red : 0.5,
+        green : 0.5,
+        blue : 0.7,
+        alpha : 0.5
+    }]
+};
+
+var scratchCartesian3 = new Cesium.Cartesian3();
+var scratchCartographic = new Cesium.Cartographic();
+var forceFunction = function(options, iteration) {
+    var iterationOffset = iteration;
+    var func = function(particle) {
```

Comment:
No abbreviations in variable names, but you should just be able to return the function directly without saving it to a variable.